tests/reliability/conftest.py

```diff
@@ -0,0 +1,60 @@
+import json
```

Comment:
Please, add the copyright

Reply:
Done [2817bcc](https://github.com/wazuh/wazuh-qa/pull/2569/commits/2817bcc46d4bf34961295ce282bc2695e22b0c4f)